tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Check installation valid mode
```

Comment:
Would be better using the `fail` module with a `when` condition. This way you can specify a message explaining why is install mode required and can only be one of two values.